salt/elife-dashboard/scheduler.sls

```diff
@@ -117,6 +117,49 @@ configure-{{ app.name }}-log:
         - require:
             - file: configure-{{ app.name }}
 
+#
+# backup
+# bit of a hack for article-scheduler
+
+# separate descriptor living in different config directory so it doesn't conflict with elife-dashboard ubr config
+ubr-{{ app.name }}-db-backup:
+    file.managed:
+        - name: /etc/ubr2/elife-article-scheduler-backup.yaml
+        - source: salt://elife-dashboard/config/etc-ubr2-elife-article-scheduler-backup.yaml
+        - makedirs: true
+        - template: jinja
+        - require:
+            - {{ app.name }}-db-exists
+
+# separate UBR config living alongside regular config
+alt-ubr-config:
+    file.managed:
+        - name: /opt/ubr/alt-app.cfg
+        - source: salt://elife-dashboard/config/opt-ubr-alt-app.cfg
+        - template: jinja
+        # read-write for root only
+        - user: root
+        - group: root
+        - mode: 600
+        - require:
+            - install-ubr
+
+# 11:15pm every day
+{{ app.name }}-daily-backups:
+    # only backup prod, adhoc and continuumtest instances
+    {% if pillar.elife.env in ['dev', 'ci', 'end2end'] %}
```

Comment:
maybe we could leave `end2end` in for consistency. S3 doesn't have space problems and databases should be very small there. We do RDS snapshots there.

Reply:
I'll fix that up then